modules/codegen/src/main/scala/com/twilio/guardrail/terms/SwaggerTerm.scala

```diff
@@ -29,7 +32,16 @@ object RouteMeta {
     }
   }
 }
-case class RouteMeta(path: String, method: HttpMethod, operation: Operation) {
+
+object SecurityRequirements {
+  type SecurityScopes = List[String]
+
+  def apply(requirements: util.List[SecurityRequirement], optionalSchemes: List[String], isGlobal: Boolean): SecurityRequirements =
```

Comment:
I'm seeing `isGlobal = true` and `isGlobal = false`. Initially I thought that was to override default parameters, but now I see it's for clarity. I'd request `IsGlobal` defined as a `sealed trait` with `Global` and `Local` (or `NonGlobal`, or something else) members, explicitly with the intent to disambiguate.

Reply:
Thanks!